joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
 
-Two other general-purpose libraries for recording phylogenetic information do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
-However, they allow different modes of phylogenetic tracking than Phylotrack does.
+Two other general-purpose libraries for phylogenetic tracking do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
+However, they pursue alternate modes of record keeping to Phylotrack.
 Whereas Phylotrack uses a graph-based approach to perfectly record asexual phylogenies, the hstrat library implements hereditary stratigraphy, a recently developed method that allows robust decentralized phylogenetic tracking in parallel and distributed systems at the cost of a tunable reduction in accuracy [@moreno2022hstrat].
 APOGeT, in turn, focuses on tracking speciation in sexually-reproducing populations [@godin2019apoget].
 
 Vast amounts of bioinformatics-oriented phylogenetics software is also available.
 These programs' purposes typically include
-- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic]
-- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating]
-- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic]
-- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython]
+- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic],
+- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating],
+- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic], and
+- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython].
 
 Phylotrack overlaps with these goals only in that it also provides tree statistic implementations. We chose to include this feature to facilitate fast during-simulation calculations of these metrics.
-Notably, the problem of tracking a phylogeny within an agent-based program is substantially different from the more traditional problem of reconstructing a phylogeny
-(indeed, users new to working with recorded phylogenies should refer to the Phylotrackpy documentation for notes on subtle structural differences from reconstructed phylogenies).
+Notably, the problem of tracking a phylogeny within an agent-based program is substantially different from the more traditional problem of reconstructing a phylogeny.
+(Indeed, users new to working with recorded phylogenies should refer to the Phylotrackpy documentation for notes on subtle structural differences from reconstructed phylogenies.)
 
-Phylotrack has already facilitated several research projects.
+Phylotrack has contributed to a variety of published research.
```

Comment:
"enabled a variety of published research projects"? For some reason my brain really wants "research" to be an adjective not a noun here.

Reply:
"tapped"? "made use of"? 

"Several published research projects have tapped/made use of Phylotrack."

Reply:
Oh oops I think I got confused about which sentence I was commenting on here - I'm fine with "contributed to." I just wanted to add "projects" to the end.

Reply:
Good with me!